enacts/wat_bal/maproom_monit.py

```diff
@@ -336,13 +336,18 @@ def wat_bal_ts(
         p_d.dt.year == planting_year, drop=True
     )).squeeze(drop=True).rename("p_d")
     precip = precip.where(
-        (precip["T"] >= p_d) & (precip["T"] < (p_d + np.timedelta64(365, "D"))),
+        (precip["T"] >= p_d - np.timedelta64(7 - 1, "D"))
```

Comment:
Why did you add this timedelta shift?

Reply:
Because I introduced the "runoff" part of the process which is based on 6-day rolling window applied to the precip so one needs to keep 6 days of precip prior to planting date (p_d) to make sure effective precipitation exists for planting.

Reply:
use a variable `window = 7` instead of hard-coding the literal 7 in multiple places.

Reply:
Done... might be factored out and more graciously handled if factoring out can be expanded.